src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -240,8 +240,14 @@ def handle_update_case_command(
         case_resolution_reason_select(optional=True),
         resolution_input(initial_value=case.resolution),
         assignee_select(initial_user=assignee_initial_user),
-        case_status_select(initial_option={"text": case.status, "value": case.status}, statuses=statuses),
-        Context(elements=["Cases cannot be escalated here. Please use the `/dispatch-escalate-case` slash command."]),
+        case_status_select(
+            initial_option={"text": case.status, "value": case.status}, statuses=statuses
+        ),
+        Context(
+            elements=[
+                "Cases cannot be escalated here. Please use the `/dispatch-escalate-case` slash command."
```

Comment:
I wonder if we could do `SlackConversationConfiguration().slack_command_escalate_case` instead of hardcoding the name of the command.